easybuild/easyconfigs/t/TensorFlow/TensorFlow_2.15.1_ARM_CUDA.patch

```diff
@@ -0,0 +1,56 @@
+diff -ruN a/tensorflow-2.15.1/third_party/absl/absl.eb b/tensorflow-2.15.1/third_party/absl/absl.eb
+--- a/tensorflow-2.15.1/third_party/absl/absl.eb	1970-01-01 01:00:00.000000000 +0100
++++ b/tensorflow-2.15.1/third_party/absl/absl.eb	2025-05-21 09:48:59.000000000 +0200
+@@ -0,0 +1,24 @@
++easyblock = 'Bundle'
++
++name = 'absl-py'
++version = '2.2.2'
++
++homepage = 'https://github.com/abseil/abseil-py'
++description = """absl-py is a collection of Python library code for building Python
++applications. The code is collected from Google's own Python code base, and has
++been extensively tested and used in production."""
++
++toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
++
++dependencies = [('Python', '3.11.3')]
++
++exts_list = [
++    ('absl-py', version, {
++        'modulename': 'absl',
++        'source_tmpl': '%(name)s-%(version)s.tar.gz',
++        'source_urls': ['https://files.pythonhosted.org/packages/source/a/absl-py/'],
++        # You can also add the checksum once verified
++    }),
++]
++
++moduleclass = 'tools'
+diff -ruN a/tensorflow-2.15.1/third_party/absl/absl_combined_fix.patch b/tensorflow-2.15.1/third_party/absl/absl_combined_fix.patch
+--- a/tensorflow-2.15.1/third_party/absl/absl_combined_fix.patch	1970-01-01 01:00:00.000000000 +0100
++++ b/tensorflow-2.15.1/third_party/absl/absl_combined_fix.patch	2025-05-22 09:43:35.000000000 +0200
+@@ -0,0 +1,11 @@
++--- a/absl/base/config.h	2023-03-27 16:58:21.000000000 +0200
+++++ b/absl/base/config.h	2025-05-21 11:31:45.501755010 +0200
++@@ -936,7 +936,7 @@
++ // https://llvm.org/docs/CompileCudaWithLLVM.html#detecting-clang-vs-nvcc-from-code
++ #ifdef ABSL_INTERNAL_HAVE_ARM_NEON
++ #error ABSL_INTERNAL_HAVE_ARM_NEON cannot be directly set
++-#elif defined(__ARM_NEON) && !defined(__CUDA_ARCH__)
+++#elif defined(__ARM_NEON) && !defined(__CUDACC__)
```

Comment:
Where exactly is this change from? Did you got and/or report it to Abseil upstream? If so please add a link to the top of this file

Also add a description and author to the top.